example/Zestfile

```diff
@@ -97,6 +97,13 @@ Version() {
 	git describe --tag 2> /dev/null
 }
 
+# Revision()
+# The full git commit hash or something equivalent
+# Used for feeding the VERSION variable into the container
```

Comment:
Typo: feeding the *REVISION* variable

Reply:
😅 oopsie - will fix in 2.0.  Thanks @turbolent !